src/components/vm/filesystems/vmFilesystemsCard.jsx

```diff
@@ -63,6 +63,10 @@ export const VmFilesystemsCard = ({ connectionName, vmName, vmState, objPath, fi
                                           actionName: _("Remove"),
                                           objectName: filesystemTarget,
                                           onClose: () => setDeleteDialogProps(undefined),
+                                          actionDescription: <TextContent>
+                                              <Text component={TextVariants.p}>{_("The current memory backing configuration will not be affected.")}</Text>
+                                              <Text component={TextVariants.p}>{_("Confirm this action.")}</Text>
```

Comment:
What does "confirm this action" mean in this context? I don't see any additional confirmation step that the user has to take here?

Reply:
When deleting resources we always show the 'Confirm this action' text. We do exactly the same for disks, nics etc.